multiply-array/cmd/test-multiply-array/main.go

```diff
@@ -55,8 +57,11 @@ func main() {
 	// Read the results into our output array and then print them out
 	binary.Read(outputBuff.Reader(), binary.LittleEndian, &output)
 
-	for _, val := range output {
-		print(val)
-	}
-
+	log.Println()
+	log.Println()
+	log.Printf("This is our input to the FPGA:")
```

Comment:
check you get the right output here, to my eye these need to end in a `\n` newline character.